### PR TITLE
Fix generators

### PR DESCRIPTION
**Pull request by jurekbarth**
#278

### CODE REVIEW

**Comment by mxstbr** [comment]
Amazing, this looks good at first glance! Thanks so much!! I'll check it out from my laptop asap, currently on mobile.

File: internals/generators/route/routeWithReducer.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (mxstbr) [inline internals/generators/route/routeWithReducer.hbs]: nit: needs a space before this

**Comment by mxstbr** [comment]
This is awesome and works exactly as intended, awesome!! :+1: :+1: 

One small nitpick before we merge this, the template needs to have 1 (one) space before each line, otherwise the indentation is a bit messed up:

screen shot 2016-04-28 at 10 30 20

**Comment by jurekbarth** [comment]
Should be fixed now :)

**Comment by mxstbr** [comment]
Amazing, thanks so much for the quick reply and for fixing this!!

**Pull request merged**